serverless/sql-databases/how-to/add-users.mdx

```diff
@@ -0,0 +1,64 @@
+---
+meta:
+  title: How to add users to a Serverless SQL Database
+  description: This page explains how to give users access to a Serverless SQL Database
+content:
+  h1: How to add users to a Serverless SQL Database
+  paragraph: This page explains how to give users access to a Serverless SQL Database
+tags: sql-databases serverless database iam postgresql
+dates:
+  validation: 2023-07-26
+  posted: 2019-07-26
+categories:
+  - serverless
+  - sql-databases
+---
+
+Users can connect to a database and access its data. Each one can have a different set of customizable permissions.
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a Serverless SQL Database](/serverless/sql-databases/how-to/create-a-database/)
+  - You have [created a policy to manage users' permissions](serverless/sql-databases/how-to/manage-permissions/)
+  - You have [invited a user to your organization](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/invite-user-to-orga/) or [created an application](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-application/) in IAM
+</Message>
+
+## How to add an existing user to a Serverless SQL Database
```

Comment:
Maybe explain what is meant by "existing user" - presumably someone who already has a SCW account? What if the person wants to add a non-existing user?